crates/ln-dlc-node/examples/fund.rs

```diff
@@ -0,0 +1,223 @@
+extern crate ln_dlc_node;
+use anyhow::bail;
+use anyhow::Context;
+use anyhow::Result;
+use bitcoin::Amount;
+use ln_dlc_node::node::NodeInfo;
+use local_ip_address::local_ip;
+use reqwest::Response;
+use serde::Deserialize;
+use std::time::Duration;
+use tracing::metadata::LevelFilter;
+use tracing_subscriber::filter::Directive;
+use tracing_subscriber::layer::SubscriberExt;
+use tracing_subscriber::util::SubscriberInitExt;
+use tracing_subscriber::EnvFilter;
+
+const RUST_LOG_ENV: &str = "RUST_LOG";
+const FAUCET: &str = "http://localhost:8080";
+const COORDINATOR: &str = "http://localhost:8000";
+
+#[tokio::main]
+async fn main() {
+    init_tracing(LevelFilter::DEBUG).expect("tracing to initialise");
+    fund_everything().await.expect("to be able to fund");
+}
+
+async fn fund_everything() -> Result<()> {
+    let coord_addr = get_coordinator_address().await?;
+    fund(&coord_addr, Amount::ONE_BTC).await?;
+    mine(20).await?;
```

Comment:
Why do you mine 20 blocks?  Isn't this waiting uneccesary long (20*5 seconds)?

Reply:
it only waits for `5` seconds, it is equivalent to multiple clicks.

20 is an arbitrary large number, might reduce that later on.